trade-finance-manager-api/src/v1/controllers/update-portal-deal-from-MIA-to-MIN.js

```diff
@@ -2,21 +2,36 @@ const CONSTANTS = require('../../constants');
 const api = require('../api');
 const now = require('../../now');
 
-const updatePortalDealFromMIAtoMIN = async (dealId, checker) => {
+const updatePortalDealFromMIAtoMIN = async (dealId, dealType, checker) => {
   console.log('Updating Portal deal from MIA to MIN');
+  let update;
+  let dealUpdate;
 
-  const dealUpdate = {
-    submissionType: CONSTANTS.DEALS.SUBMISSION_TYPE.MIN,
-    details: {
+  if (dealType === CONSTANTS.DEALS.DEAL_TYPE.GEF) {
+    dealUpdate = {
+      submissionType: CONSTANTS.DEALS.SUBMISSION_TYPE.MIN,
```

Comment:
Same as above here - if we add a test in that second submit API test like for BSS, this new condition will have test coverage 👍